Performance/runners/swift.sh

```diff
@@ -16,6 +16,14 @@
 #
 # -----------------------------------------------------------------------------
 
+# Use xcrun if it's present (macOS) or omit it if it's not (Linux)
+XCRUN=`which xcrun`
+if [ -n "${XCRUN:-}" ]; then
+    XCRUN="${XCRUN} -sdk macosx"
+fi
+# How to run 'swiftc': Use SWIFT_EXEC if it's set or just plain "swiftc"
+SWIFTC=${SWIFT_EXEC:-swiftc}
```

Comment:
The swiftc handing is different between the two scripts, there a reason for that or should it be the same?

Reply:
It's for two different programs:  `swift` in one and `swiftc` in the other.  The `SWIFT_EXEC` variable specifies the full path to `swiftc`.  To get the `swift` command, we look in the same directory for a different filename.

Someone else pointed out that as written, these scripts don't handle spaces in pathnames.  I'll take a quick stab at fixing that.

Reply:
The spaces will be harder, you'll likely need to use a bash array since you want XCRUN to pick up arguments, etc.

Reply:
After thinking about it a bit, I don't think we need to worry much about `XCRUN`, since it's very unlikely that anyone will have that anywhere other than the standard `/usr/bin/xcrun`.  The `XCRUN` variable is really only to handle macOS (has it) vs. Linux (doesn't have it).  So I really only need to worry about space-containing paths for `SWIFT` and `SWIFTC`, which is a lot simpler.